src/image_embedding/impl.rs

```diff
@@ -132,6 +132,97 @@ impl ImageEmbedding {
             .expect("Model not found.")
     }
 
+    /// Method to generate image embeddings for a Vec of image bytes
+    pub fn embed_bytes(
+        &self,
+        images: &[&[u8]],
```

Comment:
A `Vec<&[u8]` would be better?

Reply:
Would it? We can pass in a Vec just by borrowing it, but if this required a Vec, if you only had a slice, you'd have to clone it (which is expensive and may use a lot of RAM if you have a lot of images)

Reply:
True 👍